packages/netlify-cms-lib-auth/src/netlify-auth.js

```diff
@@ -126,16 +126,7 @@ class Authenticator {
     this.authWindow = window.open(
       url,
       'Netlify Authorization',
-      'toolbar=no, location=no, directories=no, status=no, menubar=no, scrollbars=no, resizable=no, copyhistory=no, ' +
-        ('width=' +
-          conf.width +
-          ', height=' +
-          conf.height +
-          ', top=' +
-          top +
-          ', left=' +
-          left +
-          ');'),
+      `width=${conf.width}, height=${conf.height}, top=${top}, left=${left}`,
```

Comment:
Any reason not to continue stripping down the auth window and just stop setting `scrollbars` to `no`? It's pretty typical for auth windows to be featureless.

Reply:
Everything is `no` by default, except things which you can't override anyway or the browser manufacturer thinks should be on. So the dropping of these other items is simply a separate cleanup, and could have been a separate commit.

Reply:
Ah, I did not consider whether those were the defaults, got it 👍